image/utils/meta_image.go

```diff
@@ -45,11 +45,17 @@ func DeleteImage(imageName string) error {
 		return err
 	}
 
-	_, ok := imagesMap[imageName]
+	imageMetadata, ok := imagesMap[imageName]
 	if !ok {
 		return nil
 	}
 	delete(imagesMap, imageName)
+	// delete image name alias records
+	for imageName, value := range imagesMap {
```

Comment:
should we delete all images with same id?

Reply:
same as docker rmi is fine